udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -390,35 +404,57 @@ <h3 class="fr-h6 fr-mb-3v fr-mt-9v">{{ type_label }} <sup>{{resources|length }}<
 {% block community %}
 <section class="community_container fr-pb-7w" id="community">
     <div class="container">
-        <h2 id="community-resources" class="fr-h2">{{ _('Community resources') }}
-            <sup>{{ dataset.community_resources|length }}</sup></h2>
-        <div class="row fr-mb-3w">
-            <div class="col-6 col-md-12 text-grey-380">
-                <p>{{ _('You have built a more comprehensive database than those presented here? This is the time to share it!') }}
-                </p>
-            </div>
-        </div>
-
-        {% if not read_only_mode %}
-        <div class="resource-list">
-            {% for resource in dataset.community_resources %}
-            {% include theme('dataset/resource/card.html') %}
-            {% endfor %}
+        <div class="fr-grid-row">
+            <h2 id="community-resources" class="fr-h2 fr-col fr-mb-5v">{{ _('Community resources') }}
```

Comment:
Do you find it weird ? Both words were kept on purpose : https://www.figma.com/file/kfpGHDywdXEr0phdHfjTMc?node-id=808:5761#150230825

Reply:
Saw your comment on Figma @maudetes. Indeed we kept *ressources* for community. 
It aims to differentiate further both objects and to express the fact that it can be something else than a "file".
The logic is that it is more important to make sure users understand where the data is whereas community ressources are a "power user" feature.